app/packages/core/src/Dataset.ts

```diff
@@ -67,6 +67,15 @@ export const DatasetNodeQuery = graphql`
         description
         info
       }
+      datasetSchema {
```

Comment:
If a dataset schema is always required, the dataset field can be queried twice with different variables (e.g. the view bar view, and the empty view). This may not be necessary
* https://stackoverflow.com/questions/50462944/is-it-possible-to-query-the-same-field-multiple-times-with-graphql